packages/apps/types/src/serializer/space-properties.ts

```diff
@@ -14,6 +14,8 @@ import { type Space } from '@dxos/react-client/echo';
  * when working with space properties.
  */
 
+// TODO(burdon): Factor out.
```

Comment:
this is mainly need for backwards compat with lodash-style nested selectors. If we migrate data stored with those kinds of selectors (mainly space properties) then we could probably just remove this util altogether